boards/shields/mcp251xfd/doc/index.rst

```diff
@@ -0,0 +1,171 @@
+.. _mcp251xFD_shield:
+
+Microchip MCP2517FD/MCP2518FD CAN bus shields
```

Comment:
unless you're reasonably confident there are/will be other shields using the same chip, it would vote for putting in a folder just named after the shield and have the doc page just focus on that shield
cc @henrikbrixandersen